### DISCUSSION TITLE
Need help to build a project

### DISCUSSION THREAD

**Discussion by VinodTarsariya** [serenity-is/Serenity#5923] [Help] [answered]
Hi Teams,

I am getting below error while building a project. I searched on google but didn't found any solution. Can any one help me for this?

![image](https://user-images.githubusercontent.com/11170564/129478518-d1233c4e-2bb8-4fe4-a6af-16d16afbd052.png)

This project is build in .Net 5.0 framework.

**Comment by VictorTomaili** [comment, marked as answer]
`dotnet tool install --global sergen`

**Reply by VinodTarsariya** [reply]
Thanks a lot. After spending so much time, you saved my time.
